### PR TITLE
adding form style to integrations

### PR DESCRIPTION
**Pull request by iruzevic**
- added meta attribute to blocks
- added additional content filter to all relevant blocks

### CODE REVIEW

File: src/Blocks/components/checkboxes/components/checkboxes-editor.js [hunk 1]
```diff
@@ -28,11 +30,28 @@ export const CheckboxesEditor = (attributes) => {
...
+			<div dangerouslySetInnerHTML={{__html: additionalContent}} />
+		</>
+	);
+
 	return (
 		<>
 			<FieldEditor
 				{...props('field', attributes, {
-					fieldContent: checkboxesContent
+					fieldContent: checkboxes
```
Comment (volfkarlo) [inline src/Blocks/components/checkboxes/components/checkboxes-editor.js:54]: Trailing comma.
```suggestion
					fieldContent: checkboxes,
```

**Review by volfkarlo — CHANGES_REQUESTED** [review]
Minor change. 
Awesome job! 😄

File: src/Helpers/Helper.php [hunk 2]
```diff
@@ -219,4 +219,63 @@ public static function isSettingsPage(): bool
...
 	}
+
+	/**
+	 * Minify string
+	 *
+	 * @param string $string String to check.
+	 *
+	 * @return string
+	 */
+	public static function minifiString(string $string): string
```
Comment (iobrado) [inline src/Helpers/Helper.php:230]: Typo?

**Review by iobrado — CHANGES_REQUESTED** [review]
Great job! 🙂  Left one question. Looks like there's a typo in one method's name.

File: src/Helpers/Helper.php (hunk 2 above)
Comment (iruzevic) [reply src/Helpers/Helper.php:230]: yap

Comment (iruzevic) [reply src/Helpers/Helper.php:230]: fixed

**Review by iobrado — APPROVED** [review]

**Pull request merged**